src/runtime/GlobalObjectBuiltinSymbol.cpp

```diff
@@ -116,6 +138,8 @@ void GlobalObject::installSymbol(ExecutionState& state)
 
     m_symbolPrototype = new Object(state);
     m_symbolPrototype->setGlobalIntrinsicObject(state, true);
+    m_symbolPrototype->markThisObjectDontNeedStructureTransitionTable();
+    m_symbolPrototype->setPrototype(state, m_objectPrototype);
```

Comment:
These two added lines are unnecessary since `setGlobalIntrinsicObject` calls `markThisObjectDontNeedStructureTransitionTable` inside itself and `m_objectPrototype` is already initialized when default `Object` constructor called.